packages/api-docs/src/rendering.ts

```diff
@@ -1,7 +1,7 @@
 import {
   DocExcerpt,
   DocInlineTag,
-  DocNodeKind,
+  // DocNodeKind,
```

Comment:
are these commented out lines intentional?

Reply:
no, I forgot to remove them, ty!